app/components/exp-lookit-iframe/doc.rst

```diff
@@ -97,16 +93,21 @@ iframeHeight [String | ``700px``]
 iframeWidth [String | ``100%``]
     Set the width of the iframe. You can use CSS units, including percents ("700px", "4in", "100%").
 
-optionalText [String]
-    Add a message underneath the iframe to contextualize what's being displayed. For instance, you can tell the participant how they 
+instructionText [String]
+    Add a message above the iframe to contextualize what's being displayed. For instance, you can tell the participant how they 
     will know when to click the Next button.
 
 optionalExternalLink [Boolean | ``false``]
     Allow participants to click on a link to open the external URL in a new tab if the iframe doesn't load correctly. This 
-    message displays under any optionalText string and reads "If you don't see anything in the space above, there might have 
+    message displays under the iframe and reads "If you don't see anything in the space above, there might have 
     been a problem loading this part of the study. Click [here] to open this part of the study in a new tab. Make sure to keep 
     this tab open so you can continue to the rest of the study."
 
+nextButtonText [String | ``Next`` ]
+    Text to display on the 'next frame' button.
+
+warningMessageText [String | ``Please confirm that you have finished the task above! When you have finished, click the button to continue.``]
+    Red text displayed above next button to confirm that the user understands that there's  a task above to be completed before moving to next frame.
```

Comment:
@okaycj Can you add a line here that says "By default, the warning message will read "Please confirm that you have finished the task above! When you have finished, click the button to continue.". Specifying specific text for this parameter will overwrite the default message with a custom message". Or something similar?

Reply:
The text in backticks after `[ String | ` is meant to indicate the parameter's default value, so I don't think we need to repeat in the description. We could mention the the accidental click thing here too (in addition to elsewhere). So how about this for the full description?

>  Red text displayed above next button to confirm that the user understands that there's a task above to be completed before moving to next frame. If no value is given, the default text (shown above) will be used, otherwise you can provide a custom message. This message will appear after the user first clicks the 'Next' button, at which point the 'Next' button will be briefly disabled to encourage users to check that they've finished the iframe task and are clicking the correct button." 

We could also consider adding this to the example, to show users how it can be used, e.g.
```json
"study-survey": {
    "kind": "exp-lookit-iframe",
    "iframeSrc": "https://example.com",
    "iframeHeight": "700px",
    "iframeWidth": "100%",
    "instructionText": "Please complete this survey. When you are finished, click the green 'Next' button at the bottom of the page.",
    "warningMessageText": "Please confirm that you have finished the survey above before continuing to the next part of the study. You should see a screen that says 'Thank you, your response has been recorded'."
}
```

Reply:
@becky-gilbert Yes I think that is a good solution!